pipestat/html_reports_pipestat.py

```diff
@@ -937,12 +945,20 @@ def get_file_for_project(prj, pipeline_name, appendix=None, directory=None, repo
     Format of the output path:
     {output_dir}/{directory}/{p.name}_{pipeline_name}_{active_amendments}_{appendix}
 
-    :param looper.Project prj: project object
+    :param pipestat manager object prj: project object
     :param str pipeline_name: name of the pipeline to get the file for
     :param str appendix: the appendix of the file to create the path for,
         like 'objs_summary.tsv' for objects summary file
-    :return str: path to the file
+    :param str directory: subdirectory
```

Comment:
add detail here. subdir for what?

Reply:
:param str directory: optional subdirectory for location of file